src/com/oltpbenchmark/benchmarks/tpch/util/CopyUtil.java

```diff
@@ -1,207 +0,0 @@
-/******************************************************************************
- * Copyright 2015 by OLTPBenchmark Project                                   *
- * *
- * Licensed under the Apache License, Version 2.0 (the "License");           *
- * you may not use this file except in compliance with the License.          *
- * You may obtain a copy of the License at                                   *
- * *
- * http://www.apache.org/licenses/LICENSE-2.0                              *
- * *
- * Unless required by applicable law or agreed to in writing, software       *
- * distributed under the License is distributed on an "AS IS" BASIS,         *
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  *
- * See the License for the specific language governing permissions and       *
- * limitations under the License.                                            *
- ******************************************************************************/
-
-package com.oltpbenchmark.benchmarks.tpch.util;
-
-import com.oltpbenchmark.WorkloadConfiguration;
-import org.apache.log4j.Logger;
-import org.postgresql.copy.CopyManager;
-import org.postgresql.core.BaseConnection;
-
-import java.io.FileInputStream;
-import java.io.FilterInputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.sql.Connection;
-import java.sql.SQLException;
-import java.util.Arrays;
-
-/**
- * copyDATABASE methods return null if they weren't successful, and return
- * a possibly empty String[] of SQL statements to execute otherwise
- */
-public class CopyUtil {
```

Comment:
I might be biased because I may have been involved in adding this, but the difference between COPY and regular old INSERT is quite high for loading. Might want to bring this back.